src/main/kotlin/com/exactpro/th2/common/utils/event/MessageBatcherDirection.kt

```diff
@@ -0,0 +1,68 @@
+package com.exactpro.th2.common.utils.event
+
+import com.exactpro.th2.common.grpc.AnyMessage
+import com.exactpro.th2.common.grpc.Direction
+import com.exactpro.th2.common.grpc.Message
+import com.exactpro.th2.common.grpc.MessageGroup
+import com.exactpro.th2.common.grpc.MessageGroupBatch
+import com.exactpro.th2.common.grpc.RawMessage
+import java.util.concurrent.CompletableFuture
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.Future
+import java.util.concurrent.ScheduledExecutorService
+import java.util.concurrent.TimeUnit.MILLISECONDS
+import java.util.concurrent.locks.ReentrantLock
+import kotlin.concurrent.withLock
+
+class MessageBatcherDirection(
```

Comment:
Do we really need a separate class for this? Why not just add batching by direction to `MessageBatcher`?